packages/web-components/src/horizontalbarchart/horizontalbarchart.ts

```diff
@@ -115,6 +118,26 @@ export class HorizontalBarChart extends FASTElement {
       },
     ];
 
+    const allchartPoints = [...chartPoints1, ...chartPoints2, ...chartPoints3];
```

Comment:
these should be part of the story not the .ts file

Reply:
Yes for now, we have defined data also in .ts file only. Moving forward when will shift that to the stories file, then we can move this as well.